tasks/fr/fr_income.py

```diff
@@ -144,12 +143,10 @@ def columns(self):
 
         return cols
 
-    def get_targets(self, cols, denominators, universes):
+    def get_targets(self, cols, denominators):
         targets_dict = {}
         if denominators:
             targets_dict = self.get_relation_targets(cols, denominators, DENOMINATOR)
```

Comment:
`get_relation_targets` was useful when we had both denominators and universes. Since we only have denominators now, we may consider removing the method